docs/csharp/language-reference/builtin-types/built-in-types.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Built-in types - C# reference"
+description: "Learn C# built-in value and reference types"
+ms.date: 02/04/2020
+helpviewer_keywords: 
+  - "types [C#], built-in"
+  - "built-in C# types"
+ms.assetid: 54f901f2-bf2f-472c-ae8d-73e8ecfc57fe
+---
+# Built-in types (C# reference)
+
+The following table lists the C# built-in [value](value-types.md) types:
+
+|C# type keyword|.NET type|
+|--------------|-------------------------|
+|[bool](bool.md)|<xref:System.Boolean?displayProperty=nameWithType>|
+|[byte](integral-numeric-types.md)|<xref:System.Byte?displayProperty=nameWithType>|
+|[sbyte](integral-numeric-types.md)|<xref:System.SByte?displayProperty=nameWithType>|
+|[char](char.md)|<xref:System.Char?displayProperty=nameWithType>|
+|[decimal](floating-point-numeric-types.md)|<xref:System.Decimal?displayProperty=nameWithType>|
+|[double](floating-point-numeric-types.md)|<xref:System.Double?displayProperty=nameWithType>|
+|[float](floating-point-numeric-types.md)|<xref:System.Single?displayProperty=nameWithType>|
+|[int](integral-numeric-types.md)|<xref:System.Int32?displayProperty=nameWithType>|
+|[uint](integral-numeric-types.md)|<xref:System.UInt32?displayProperty=nameWithType>|
+|[long](integral-numeric-types.md)|<xref:System.Int64?displayProperty=nameWithType>|
+|[ulong](integral-numeric-types.md)|<xref:System.UInt64?displayProperty=nameWithType>|
+|[short](integral-numeric-types.md)|<xref:System.Int16?displayProperty=nameWithType>|
+|[ushort](integral-numeric-types.md)|<xref:System.UInt16?displayProperty=nameWithType>|
+
+The following table lists the C# built-in [reference](../keywords/reference-types.md) types:
+
+|C# type keyword|.NET type|
+|--------------|-------------------------|
+|[object](reference-types.md#the-object-type)|<xref:System.Object?displayProperty=nameWithType>|
```

Comment:
Same (adding comment because of two tables)

```suggestion
|[`object`](reference-types.md#the-object-type)|<xref:System.Object?displayProperty=nameWithType>|
```